src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v2-monitoring-integration.mdx

```diff
@@ -0,0 +1,151 @@
+
+## Activate integration [#activate-integration]
+
+To enable this integration, see how to [connect AWS services to New Relic](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/).
+
+## Find and use data [#find-use-data]
+
+To find your integration's metrics, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Metrics & Events** and filter by `aws.apigateway`.
+
+## Metric data [#metric-data]
+
+This New Relic infrastructure integration collects the following Amazon API Gateway v2 metrics:
+
+### Amazon API Gateway v2 Metric data
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+      Metric  
+      </th>
+      <th>
+      Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+      `4xx`
+      </td>
+      <td>
+      The number of client-side errors captured in a given period.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `5xx`
+      </td>
+      <td>
+      The number of server-side errors captured in a given period.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Count`
+      </td>
+      <td>
+      The total number API requests in a given period.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `IntegrationLatency`
+      </td>
+      <td>
+      The time between when API Gateway relays a request to the backend and when it receives a response from the backend.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Latency`
+      </td>
+      <td>
+      The time between when API Gateway receives a request from a client and when it returns a response to the client. The latency includes the integration latency and other API Gateway overhead.
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `DataProcessed`
+      </td>
+      <td>
+      The amount of data processed in bytes.
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+## Dimensions
+
+You can use the [dimensions](http://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_Dimension.html) in the following table to filter API Gateway metrics.
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "285x" }}>
+        Dimensions
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `ApiId`
+      </td>
+
+      <td>
+        ilters API Gateway metrics for an API with the specified API ID.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        `ApiId`, `Stage`
+      </td>
+
+      <td>
+        Filters API Gateway metrics for an API stage with the specified API ID and stage ID.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        `ApiId`, `Stage`,`Route`
+      </td>
+
+      <td>
+        Filters API Gateway metrics for an API method with the specified API ID, stage ID, and route ID.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+### Create alerts
```

Comment:
```suggestion
### Create alerts [#create-alerts]
```

Reply:
@rhetoric101, I made the suggested change.